src/tools/miri/src/shims/tls.rs

```diff
@@ -322,6 +322,7 @@ trait EvalContextPrivExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         // but both are ignored by std.
         this.call_function(
             thread_callback,
+            // TODO: is there any equivalent Conv:: that we can use here? 
             ExternAbi::System { unwind: false },
```

Comment:
@RalfJung The ``Conv`` equivalent [here](https://github.com/rust-lang/rust/blob/948cec0fad6d87f7f5993517f913aa0454b78674/compiler/rustc_ty_utils/src/abi.rs#L306) is ``bug!("system abi should be selected elsewhere")``, is there any workaround for this?

Reply:
Well you have to find where the System ABI is chosen. :) Please go grepping and searching through the rustc codebase. I can't always do this for you.

Here is a hint:

https://github.com/rust-lang/rust/blob/fb00adbdb69266f10df95a4527b767b0ad35ea48/compiler/rustc_target/src/spec/mod.rs#L2766-L2768

Reply:
Alright, thanks for the hint. I initially thought it is a bug to convert `` ExternAbi::System `` to ``Conv``, I figured out what is going on there now. :)